extra-requirements.txt

```diff
@@ -3,6 +3,7 @@ matplotlib ~= 3.2.1
 scipy ~= 1.4.1
 sympy ~= 1.6.1
 torch ~= 1.5.0
+gym
```

Comment:
0.17.0? (at least that's what I am using ^^)

Reply:
I went into some weird dependency problems, related to the pillow package. Removing the specifier made it work.